FWCore/ParameterSet/python/Config.py

```diff
@@ -151,6 +151,9 @@ def filterNames(self):
     def pathNames(self):
         """Returns a string containing all the Path names separated by a blank"""
         return ' '.join(self.paths_().keys())
+    def subProcessNames(self):
+        """Returns a string containing all the subProcess names separated by a blank"""
+        return ' '.join(self.subProcesses_().keys())
```

Comment:
I don't think this would help since you can't use this name to review the specific SubProcess.